markdown/opentelemetry-intro/opentelemetry-intro.md

```diff
@@ -0,0 +1,320 @@
+author: Jesse Menning
+summary:
+id: opentelemetry-intro
+tags:
+categories: opentelemetry, nyctaxi
+environments: Web
+status: Draft
+feedback link: <insert github repo or leave empty>
+analytics account: UA-3921398-10
+
+# Using OpenTelemetry to find your event in a haystack
+
+## What you'll learn: Overview
+
+Duration: 0:05:00
+
+
+Positive
+: If you'd like to learn more about OpenTelemetry architecture, but don't care about hands-on stuff, you can find it in the [Solace architecture blogs](https://solace.com/blog/category/architects/).
```

Comment:
I'm assuming this "Solace architecture blogs" will be replaced by the actual blog when its released?